connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -2164,6 +2423,7 @@ ble_error_t Gap::stopAdvertising(advertising_handle_t handle)
 bool Gap::isAdvertisingActive(advertising_handle_t handle)
 {
     if (handle >= getMaxAdvertisingSetNumber()) {
+        tr_error("Cannot determine if periodic advertising is active: handle greater than/equal to %d", getMaxAdvertisingSetNumber());
```

Comment:
I don't think this is specific to periodic, also a simple "invalid advertising handle %d" would be enough

Reply:
This trace does not have a proceeding `tr_info` detailing the function signature and arguments. I added the "Cannot determine if periodic advertising is active : " part to give the user a hint as to where the program erred.

Reply:
What I meant is: this is not for periodic advertising, this is normal advertising.